sentry-uitest/sentry-uitest-android-benchmark/build.gradle.kts

```diff
@@ -0,0 +1,132 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+import net.ltgt.gradle.errorprone.errorprone
+
+plugins {
+    id("com.android.application")
+    kotlin("android")
+    id(Config.QualityPlugins.errorProne)
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.QualityPlugins.detektPlugin)
+}
+
+configure<JavaPluginExtension> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+android {
+    compileSdk = Config.Android.compileSdkVersion
+
+    defaultConfig {
+        applicationId = "io.sentry.uitest.android.benchmark"
+        minSdk = Config.Android.minSdkVersionNdk
+        targetSdk = Config.Android.targetSdkVersion
+        versionCode = 1
+        versionName = "1.0.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        // Runs each test in its own instance of Instrumentation. This way they are isolated from
+        // one another and get their own Application instance.
+        // https://developer.android.com/training/testing/instrumented-tests/androidx-test-libraries/runner#enable-gradle
+        // This doesn't work on some devices with Android 11+. Clearing package data resets permissions.
+        // Check the readme for more info.
+//        testInstrumentationRunnerArguments["clearPackageData"] = "true"
+    }
+
+    testOptions {
+        execution = "ANDROIDX_TEST_ORCHESTRATOR"
+    }
+
+    signingConfigs {
+        getByName("debug") {
+            storeFile = rootProject.file("debug.keystore")
+            storePassword = "android"
+            keyAlias = "androiddebugkey"
+            keyPassword = "android"
+        }
+    }
+
+    testBuildType = System.getProperty("testBuildType", "debug")
+
+    buildTypes {
+        getByName("debug") {
+            isDebuggable = false
+            isMinifyEnabled = true
```

Comment:
```suggestion
```

Reply:
We want development version to be the same as release, so that we can run a benchmark locally with correct results.
Proguard is used in release and can affect performances, so we want to enable it in debug, too

Reply:
I know but its duplicated, you have 2 of them, isMinifyEnabled L55 and L56